packages/blockchain/CHANGELOG.md

```diff
@@ -6,7 +6,25 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## 7.2.0 - 2024-03-05
+## 7.3.0 - 2024-08-15
+
+### EIP-6110 (Deposit Requests) / EIP-7002 (Withdrawal Requests) Support
```

Comment:
Should this not include EIP-7251 also?

Reply:
Hi Jochem, thanks for the full review! 🙏 😄 

Have improved on the headline here.

(will push everything later on)